uk.co.compucorp.civicrm.hrcomments/CRM/HRComments/API/Query/CommentSelect.php

```diff
@@ -45,6 +45,8 @@ private function buildCustomQuery() {
    * @param \CRM_Utils_SQL_Select $customQuery
    */
   private function addWhere(CRM_Utils_SQL_Select $customQuery) {
+    $conditions = $this->invokeSelectWhereClauseHook();
```

Comment:
We should probably add another function to return the conditions here rather than calling the `invokeSelectWhereClauseHook` directly. Something like:

`public function getConditionsForACL() {
 return $this->invokeSelectWhereClauseHook();
}`

The reason for this is that:
(1) Its more readable
(2) In the future the `invokeSelectWhereClauseHook()` can return more than just the conditions array and may need to return other parameters.